lib/device_map.py

```diff
@@ -994,7 +995,8 @@
 x86_kvm = {'device_type': 'kvm',
            'templates': ['generic-x86-kernel-ci-boot-template.json',
                          'generic-x86-kernel-ci-kselftest-template.json',
-                         'generic-x86-kernel-ci-hackbench-template.json'],
+                         'generic-x86-kernel-ci-hackbench-template.json',
+                         'qemu-boot-template.jinja2'],
```

Comment:
I assume the .jinja2 template won't be a problem with the old job creator (I don't remember how it looks for the templates).